src/remote.js

```diff
@@ -259,20 +278,20 @@
                       util.err(error);
                     }
                   )
-                },
-                function(error) {
-                  util.err(error);
                 }
-              )
+              },
+              function(error) {
+                util.err("Unable to build this project. " + error);
+              }
+            ),
+            function(error) {
+              util.err('Unable to create monaca project: ' + error);
             }
-          },
-          function(err) {
-            util.err("Unable to build this project " + err);
           }
-        )
-      },
-      function(err) {
-        util.err(err);
+        ),
+        function(err) {
```

Comment:
Here I could not force an error, maybe Fran can check what's the type of err object and confirm the user gets an appropriate error message.